### PR TITLE
Tabs highlight

### PR DESCRIPTION
**Pull request by irenesmith**
Modified support for the changing highlighted status using the highlight method as of Firefox 63.

### CODE REVIEW

File: webextensions/api/tabs.json [hunk 1]
```diff
@@ -2917,7 +2917,7 @@
                     "version_added": false
                   },
                   "firefox": {
-                    "version_added": false
+                    "version_added": 63
```
Comment (wbamberg) [inline webextensions/api/tabs.json]: This and line 1492 need to be strings. That's why the build fails for this PR. It's useful to run "npm run lint" locally before pushing your changes, then you can catch problems like this earlier.

**Comment by irenesmith** [comment]
tabs.update.updateProperties.highlighted was an error. Should I back it out here or will that cause more problems?

If tabs.Tab.highlighted is already supported, why should it be updated again?

**Comment by wbamberg** [comment]
> If tabs.Tab.highlighted is already supported, why should it be updated again?

Because as I read it, it's inaccurate (as currently stated).

Before 63, `highlighted` was supported as an alias for `active`. From 63 it's properly supported (because from 63 Firefox supports highlighting multiple tabs). See for example the compat data for [`tabs.onHighlighted`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/onHighlighted#Browser_compatibility) - I think the compat data for `tabs.Tab.highlighted` should look similar.

Does that seem correct to you? I'm not sure, maybe this isn't necessary...

> tabs.update.updateProperties.highlighted was an error. Should I back it out here or will that cause more problems?

It would be better I think for this PR to include only the changes you want to make, unless you think this will cause more problems.

**Comment by wbamberg** [comment]
@irenesmith , please also include data for `highlightInfo` as per https://github.com/mdn/browser-compat-data/pull/2987, which I just closed in favour of this PR.

**Comment by Loirooriol** [comment]
Firefox 63 has been released and MDN still says that this is not supported. @irenesmith please fix your PR so that this can be merged or @wbamberg merge #2987 instead.

**Pull request merged**